Dockerfile

```diff
@@ -34,25 +23,18 @@ RUN wget --directory-prefix=${TEMP_DIR} ${OC_DL_URL} && \
     tar -C ${OC_BIN} -xzf ${TEMP_DIR}/${OC_BIN_TAR} && \
     chmod a+x ${OC_BIN}/oc
 
-# Add go and oc binary directory to $PATH
-ENV PATH=${PATH}:"/usr/local/go/bin":${GOPATH}/"bin"
-
-# Git identifier to checkout
-ARG TNF_VERSION
-ARG TNF_SRC_URL=$TNF_SRC_URL
-ARG GIT_CHECKOUT_TARGET=$TNF_VERSION
-
-# Clone the TNF source repository and checkout the target branch/tag/commit
-RUN git clone --no-single-branch --depth=1 ${TNF_SRC_URL} ${TNF_SRC_DIR}
-RUN git -C ${TNF_SRC_DIR} fetch origin ${GIT_CHECKOUT_TARGET}
-RUN git -C ${TNF_SRC_DIR} checkout ${GIT_CHECKOUT_TARGET}
+# Download operator-sdk binary
+ENV OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.26.0
+ENV OSDK_BIN="/usr/local/osdk/bin"
+RUN mkdir -p ${OSDK_BIN} && \
+    curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_linux_amd64 && \
```

Comment:
Should it be a tab instead of spaces as indentation?

Reply:
I am using tabs.  Is it not showing up as tabs?

Reply:
Here, at the comments, GitHub shows spaces :-)